src/config/types.js

```diff
@@ -19,7 +19,7 @@ export const YOROI_PROVIDER_IDS = {
   ALONZO_TESTNET: ALONZO_FACTOR + NETWORK_REGISTRY.HASKELL_SHELLEY_TESTNET,
 }
 
-export type YoroiProvider = 'emurgo-shelley' | 'emurgo-alonzo'
+export type YoroiProvider = '' | 'emurgo-alonzo'
```

Comment:
what does this do? what does this change do?

Reply:
since all shelley wallets wont have this property, is easier to consider '' as shelley, otherwise we have to migrate the store.